src/atoms/Link/Link.js

```diff
@@ -46,25 +46,24 @@ export default function Link({
   //If it does, append it to the end our current pages url for "to".
   const isAnchor = /^[#]/.test(linkHref);
 
-  if (isAnchor && typeof window !== "undefined") {
+  if (isAnchor && typeof window !== 'undefined') {
     return (
       <GatsbyLink
         to={`${window.location.pathname}${linkHref}`}
-        sx={{ variant: "styles.a" }}
+        sx={{ variant: 'styles.a' }}
         onClick={() => {
-          //TODO(Rejon): See how marketing team wants to handle anchor links.
-          // if (onClick !== null && onClick !== undefined) {
-          //   onClick();
-          // }
-          // const eventProps = Object.assign(
-          //   {
-          //     category: "Internal Link",
-          //     action: "Click",
-          //     label: linkHref,
-          //   },
-          //   gaProps
-          // );
-          // trackCustomEvent(eventProps);
+          if (onClick !== null && onClick !== undefined) {
+            onClick();
+          }
+          const eventProps = Object.assign(
```

Comment:
Since we are using ES6, better to do:
```
const eventProps = {
  category: 'Internal Link',
  action: 'Click',
  label: linkHref,
  ...gaProps
}
```